From c124f0c2fc47c882391160baa895161fc6356180 Mon Sep 17 00:00:00 2001
From: Ward Poelmans <wpoely86@gmail.com>
Date: Mon, 7 Oct 2013 13:32:08 +0200
Subject: [PATCH 1/2] Use configure supplied blas and lapack in the pkg-config

Use the blas and lapack library found by the configure script in the
pkg-config file instead of hardcoded libraries.
---
 arpack.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arpack.pc.in b/arpack.pc.in
index 83739bd..55d2e01 100644
--- a/arpack.pc.in
+++ b/arpack.pc.in
@@ -5,5 +5,5 @@ libdir=@libdir@
 Name: arpack
 Description: ARPACK-NG 
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -larpack -lblas -latlas
+Libs: -L${libdir} -larpack @BLAS_LIBS@ @LAPACK_LIBS@
 Cflags: 
-- 
1.8.3.1


From 29d8cbe63ffea965d46caad7773fe6dd61960dc7 Mon Sep 17 00:00:00 2001
From: Sylvestre Ledru <sylvestre.ledru@scilab-enterprises.com>
Date: Mon, 7 Oct 2013 14:25:51 +0200
Subject: [PATCH 2/2] Update of the release notes

---
 CHANGES | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/CHANGES b/CHANGES
index 996a54b..39bb865 100644
--- a/CHANGES
+++ b/CHANGES
@@ -1,4 +1,9 @@
-arpack-ng - 3.1.5
+arpack-ng - 3.1.4
+
+ * libparpack2: missing dependency on MPI:
+   http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=718790
+
+ * Replace LAPACK second function with ARPACK's own arscnd in PARPACK
 
  * Fix issue #1259 in DSEUPD and SSEUPD
    The Ritz vector purification step assumes workl(iq) still contains the
@@ -10,15 +15,10 @@ arpack-ng - 3.1.5
    space WORKL(IW+NCV:IW+2*NCV) is not used later in the routine, and can
    be used for this.
 
- -- Sylvestre Ledru <sylvestre.ledru@scilab-enterprises.com>  Thu, 29 Aug 2013 10:53:03 +0200
-
-arpack-ng - 3.1.4
-
- * libparpack2: missing dependency on MPI:
-   http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=718790
- * Replace LAPACK second function with ARPACK's own arscnd in PARPACK
+ * Use configure supplied blas and lapack in the pkg-config.
+   Thanks to Ward Poelmans (Closes: #1320)
 
- -- Sylvestre Ledru <sylvestre.ledru@scilab-enterprises.com>  Tue, 06 Aug 2013 15:15:55 +0200
+ -- Sylvestre Ledru <sylvestre.ledru@scilab-enterprises.com>  Mon, 07 Oct 2013 14:24:42 +0200
 
 arpack-ng - 3.1.3
 
-- 
1.8.3.1

